packages/super-components/components/msc-tile/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "see_more_label": "En savoir plus",
+  "product_tile_type": "Catégorie",
+  "faq_tile_type": "Tutoriel"
```

Comment:
**suggestion:** Each translation key can be prefixed with super-component name to avoid conflicts with same key being available in other super-component

Reply:
How would it conflicts ?
Translations keys are dynamically loaded by the super-component based on the language so no need to complicate the keys since each super-component comes with its own translation files.